src/libraries/Common/src/Interop/Windows/Advapi32/Interop.LsaLookupNames2.cs

```diff
@@ -9,7 +9,9 @@ internal static partial class Interop
 {
     internal static partial class Advapi32
     {
-        [DllImport(Interop.Libraries.Advapi32, EntryPoint = "LsaLookupNames2", SetLastError = true, CharSet = CharSet.Unicode)]
+#pragma warning disable DLLIMPORTGENANALYZER015 // Use 'GeneratedDllImportAttribute' instead of 'DllImportAttribute' to generate P/Invoke marshalling code at compile time
```

Comment:
Should we update the analyzer to not warn in these cases until they're fixed?

Reply:
Agree. Let me see if I can get that going in the next PR.